test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by Git
```

Comment:
- [ ] Test.md file ignored by github. I can see this file still in your repository.[Requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20test.md,your%20GitHub%20repository.)